src/main/java/teammates/ui/request/DeadlineExtensionRequest.java

```diff
@@ -0,0 +1,70 @@
+package teammates.ui.request;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The request specifying deadline extensions to create, update or revoke.
+ */
+public class DeadlineExtensionRequest extends BasicRequest {
+
+    private String courseId;
+    private String feedbackSessionName;
+    private boolean notifyUsers;
+    private Map<String, Long> studentExtensionsToModify;
+    private List<String> studentExtensionsToRevoke;
+    private Map<String, Long> instructorExtensionsToModify;
+    private List<String> instructorExtensionsToRevoke;
```

Comment:
This seems like it came from my PR, why is this necessary?

Reply:
My bad, I was picking out the requests files to generate the types. Will remove